### DISCUSSION TITLE
Tero Subtitler 1.0.0.9

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#145] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.0.9'>Tero Subtitler 1.0.0.9</a>.</em>

**Comment by andreanatile** [comment]
The installer doesn't work on my computer.

**Reply by GitGianluc** [reply]
@andreanatile could you specify which operating system you're using and which installation file is not working? Thanks!

**Reply by andreanatile** [reply]
It's all right, it was a problem with the antivirus. Btw I'm using windows.

**Reply by GitGianluc** [reply]
Fine, @URUWorks will be glad to hear it!

**Reply by URUWorks** [reply]
Nice, thanks!
